tests/core_tests/loki_tests.cpp

```diff
@@ -333,20 +364,19 @@ bool loki_checkpointing_alt_chain_with_increasing_service_node_checkpoints::gene
   // Setup the two chains as follows, where C = checkpointed block, B = normal
   // block, the main chain should NOT reorg to the fork chain as they have the
   // same PoW-ish and equal number of checkpoints.
-  // Main chain - C B B B B
-  // Fork chain - B B B B C
+  // Main chain   C  B B B B
+  //              |
+  // Fork chain   -> B B B C
```

Comment:
I'm not sure I understand the diagram change here.  It looks like the fork chain has two checkpoints and the main chain has one?

Reply:
Hmm outdated diagram. Reverted.